app/controllers/course/admin/codaveri_settings_controller.rb

```diff
@@ -10,6 +10,10 @@ def assessment
   end
 
   def update
+    head :forbidden and return if
```

Comment:
For discussion: I don't see any instances of `head :forbidden` in our codebase.  
can consider using CanCanCan to authorize instead.